www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -71,6 +71,15 @@ function (declare, domConstruct, ItemFileWriteStore, topic, Dialog, Button,
     return container;
   }
 
+  function prettifyStatus(statusCounts) {
+    var stat = [];
```

Comment:
This can be written as:
```js
return Object.keys(statusCounts).map(function (statusCount) {
  return statusCount + '&nbsp(' + statusCounts[statusCount] + ')';
}).join(', ');
```